book/cli/cli.md

```diff
@@ -64,21 +64,21 @@ Logging:
 
       --log.directory <PATH>
           The path to put log files in
-
-          [default: /Users/georgios/Library/Caches/reth/logs]
+          
+          [default: /Users/reth/Library/Caches/reth/logs]
```

Comment:
I wonder if we can fix these paths to be a bit more agnostic, I think if I were to run the makefile rn it would do `/home/oliver/.local/share..` which would trigger the CI, no?

Reply:
I have a `sed` cmd in `Makefile` to replace all `/Users/**` path into `/Users/reth` to avoid those issues.

Reply:
Note that /Users is only true for macOS, on Linux it would be /home (usually)